scripts/integration-single.sh

```diff
@@ -1,3 +1,8 @@
+# Install xdg-user-dirs for integration tests  
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    sudo apt-get install -y xdg-user-dirs
```

Comment:
Added this from our integration tests at https://github.com/OpenCerts/opencerts-website/pull/513

We're using `xdg-user-dirs` to get the downloads folder, so that we can have better integration test coverage:

- Download the file
- Check if the file contents are what we're expecting

Reply:
yeah the tricky part is we're not using `href`s anymore

thanks for the SO link! in fact, i even added on our approach for opencerts (using downloads-folder) back in June as there was no other solution

however, it seems like someone from TestCafe added this ~2 weeks ago and created an example in response to the question asked:

> You can inspect how to test file download in TestCafe using [this](https://github.com/DevExpress/testcafe-examples/blob/master/examples/check-downloaded-file-name-and-content/index.js) runnable example. – mlosev Oct 14 at 8:10

Reply:
Actually - I found a better way. Same approach, but using `os` and `path` to join the Downloads folder.